src/printing.py

```diff
@@ -10,8 +10,15 @@
 
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
+
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print('x is %d, y is %.2f, z is "%s"' % (x, y, z))
+
 # Use the 'format' string method to print the same thing
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+output = 'x is {}, y is {}, z is "{}"'
```

Comment:
This works, but there's also a way to do the rounding inside the formatted string itself like you did with the one above. Check out this link https://mkaz.blog/code/python-string-format-cookbook/